cogs/kill.py

```diff
@@ -91,7 +91,7 @@ async def kill(self, ctx: TeXBotApplicationContext) -> None:
             await ctx.respond(content="My battery is low and it's getting dark...")
             logger.info("Manual shutdown initiated by %s.", ctx.interaction.user)
             await self.bot.close()
-            return
+            sys.exit(0)
 
```

Comment:
Unfortunately `self.bot.close()` performs the exit call internally. The return statement was just below it to appease mypy. So this exit call will never be executed.

Is there a way to pass the exit code into the bot close function?
The bot close function has to be run to shut down rather than doing a raw sys.exit because the bot close function performs necessary cleanup tasks.

Reply:
do we *need* to pass an exit code to the close function? I would assume if it's closing gracefully it would use exit code 0 anyway, no?

Reply:
I'm not sure what exit code it's using for certain, However it's better to be explicit, as we cannot guarantee that pycord might change it in future and we *need* it to always be 0.